### PR TITLE
chore: Publish to gh-pages

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,34 +38,24 @@ endif
 .PHONY: all
 all: stagingrepo stablerepo
 
-.PHONY: clean
-clean:
```
Comment (makkes) [inline Makefile]: I'd prefer to stick with a `clean` target that removes the `gh-pages` dir.

File: Makefile [hunk 2]
```diff
@@ -39,34 +38,24 @@ endif
...
-	git add -A docs
-	git reset $(NON_DOCS_FILES)
-	git clean -fdx
-	git commit -m "$(LAST_COMMIT_MESSAGE)"
-	git push publish master
-	git checkout -
+	cd gh-pages && \
+		git add -A . && \
+		git commit -m "$(LAST_COMMIT_MESSAGE)" && \
+		git push --dry-run publish HEAD:gh-pages
```
Comment (makkes) [inline Makefile]:
```suggestion
		git push publish HEAD:gh-pages
```

File: Makefile (hunk 1 above)
Comment (jimmidyson) [reply Makefile]: Done.

File: Makefile (hunk 2 above)
Comment (jimmidyson) [reply Makefile]: Oh yeah! Done.

**Comment by jimmidyson** [comment]
Going to merge to give this a go..

**Pull request merged**